sdb/command.py

```diff
@@ -225,14 +234,145 @@ def __init_subclass__(cls, **kwargs):
 
     def _call(self,
               objs: Iterable[drgn.Object]) -> Optional[Iterable[drgn.Object]]:
-        # pylint: disable=missing-docstring
-        raise NotImplementedError
+        """
+        Optionally implemented by the user.
+
+        Commands that expect to manipulate and deal with any errors
+        of all the objects from the previous stage of the pipeline
+        passed as an iterable, should implement this method.
+
+        Takes precedence over _call_one() - see call() method.
+        """
+        #
+        # We should never get here!
+        #
+        assert self.name is None
+        assert False
+        return objs
+
+    def _call_one(self, obj: drgn.Object) -> Optional[drgn.Object]:
```

Comment:
Maybe this should return `Iterable[drgn.Object]`, so that it could e.g. find and yield multiple objects inside the `obj`.  That said, there are not many _call methods that yield multiple things per input object (AFAIK only the abstract classes Walker and Locator do).  In fact there are not many _call methods period, since most commands are PrettyPrinter and/or Locators.

Reply:
That was a great catch! `member` actually can return multiple objects already from one input object - when you specify the dereference of multiple fields of that struct:
```
--- a/tests/integration/data/regression_output/core/addr spa_namespace_avl | member avl_root.avl_pcb avl_size
+++ b/tests/integration/data/regression_output/core/addr spa_namespace_avl | member avl_root.avl_pcb avl_size
@@ -1 +1,2 @@
 (uintptr_t)1
+(size_t)9176
```